pytest_selenium/drivers/remote.py

```diff
@@ -9,7 +9,9 @@
 
 
 def driver_kwargs(capabilities, firefox_profile, host, port, **kwargs):
-    executor = "http://{0}:{1}/wd/hub".format(host, port)
+    executor = f"{host}:{port}/wd/hub"
```

Comment:
@platonoff-dev I know that this is totally valid and working code but what about:
```
protocol = "" if re.search(r"^https?://", host) else "http://"
executor = f"{protocol}{host}:{port}/wd/hub
```

Reply:
I'm good with that. It's one of my problems 😅. I thought about that, but decided that my solution is more kind for new developers. They faster understand my version.

Reply:
But your suggestion looks cleaner. I'll change that

Reply:
I am not sure about the regexp though ...

Reply:
```
❯ python test_performance.py
regexp: 0.009595889999999996
if: 0.004040118000000002
```
Almost two times slower...

Reply:
Code that I ran:
```python
import re
import timeit


def regexp_search():
    host = "https://some.host.com"
    port = "4444"
    protocol = "" if re.search(r"^https?://", host) else "http://"
    executor = f"{protocol}{host}:{port}/wd/hub"


def if_search():
    host = "https://some.host.com"
    port = "4444"
    executor = f"{host}:{port}/wd/hub"
    if not executor.startswith("http://") and not executor.startswith("https://"):
        executor = "http://" + executor


if __name__ == "__main__":
    print("regexp:", timeit.timeit(regexp_search, number=10000))
    print("if:", timeit.timeit(if_search, number=10000))
```

Reply:
Wow, thanks for testing that! I think that settles it then :D